.travis.yml

```diff
@@ -22,9 +29,10 @@ install:
   - cd azerothcore
   - git submodule update --init --recursive
 
+  - mysql -e "SET GLOBAL sql_mode = '';" # this is necessary because of mysql 5.7
   - mysql -uroot -e 'create database test_mysql;'
   - cd bin
-  - cmake ../ -DWITH_WARNINGS=1 -DWITH_COREDEBUG=0 -DUSE_COREPCH=1 -DUSE_SCRIPTPCH=1 -DTOOLS=0 -DSCRIPTS=1 -DSERVERS=1 -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=check_install
+  - cmake ../ -DWITH_WARNINGS=1 -DWITH_COREDEBUG=1 -DUSE_COREPCH=0 -DUSE_SCRIPTPCH=0 -DTOOLS=0 -DSCRIPTS=1 -DSERVERS=1 -DWITH_PERFTOOLS=1 -DENABLE_EXTRA_LOGS=1 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS="-Werror" -DCMAKE_CXX_FLAGS="-Werror" -DCMAKE_INSTALL_PREFIX=check_install
```

Comment:
good to disable pch, that makes me think that some modules are not compiling with PCH = 0 but I'm not sure if it's the same thing in travis

Have you tried to compile without it? just for curiosity